ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```
@@ -888,6 +888,25 @@ keys:
                     type: bool
                   inter_domain:
                     type: bool
+          next_hop_mpls_resolution_ribs:
+            type: list
+            min_length: 1
+            max_length: 3
+            description: "Specify the RIBs used to resolve MPLS next-hops. The order of this list determines the order of RIB lookups."
+            items:
+              type: dict
+              keys:
+                rib_type:
+                  type: str
+                  required: true
+                  valid_values:
+                    - "system-connected"
+                    - "tunnel-rib-colored"
+                    - "tunnel-rib"
+                  description: "Type of RIB, one of system-connect|tunnel-rib-colored|tunnel-rib. For 'tunnel-rib', use 'rib_name' to specify the name of the Tunnel-RIB to use. Use 'tunnel-rib-colored' for 'tunnel-rib colored system-colored-tunnel-rib'"
```

We document the choices under the key, so we don't have to repeat in the description.
Also I believe the colored is clear enough from the choice.

```suggestion
                  description: "Type of RIB. For 'tunnel-rib', use 'rib_name' to specify the name of the Tunnel-RIB to use."
```